src/pages/SignIn/Index.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from "react";
+import { Button, Typography, Modal } from "antd";
+import { FieldValues, useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { useNavigate } from "react-router-dom";
+import Field from "components/DefaultField/Index";
+import { useTranslation } from "react-i18next";
+import { PRIMARY } from "utils/colorConsts";
+import { PROFILE_ROUTE, SIGN_UP_ROUTE } from "utils/routeConsts";
+import S from "./style";
+import signInSchema from "./schema";
+
+interface SignInForm extends FieldValues {
+  email: string;
+  password: string;
+}
+
+export default function SignIn() {
+  const navigate = useNavigate();
+  const { t } = useTranslation();
+  const { Title, Text } = Typography;
+  const [isErrorModalOpen, setIsErrorModalOpen] = useState(false);
+  const { control, handleSubmit } = useForm<SignInForm>({
+    resolver: yupResolver(signInSchema),
+  });
+
+  const handleCloseErrorModal = () => {
+    setIsErrorModalOpen(false);
+  };
+
+  const success = () => {
+    Modal.success({
+      title: t("SignIn.successLogin"),
+      onOk: () => navigate(PROFILE_ROUTE),
+      centered: true,
+    });
+  };
+
+  const error = () => {
+    Modal.error({
+      title: t("SignIn.errorLogin"),
+      centered: true,
+      okButtonProps: { danger: true },
+    });
+  };
+
+  const goToRegister = () => {
+    window.scroll(0, 0);
```

Comment:
hmmm why its needed?

Reply:
oh I remember this bug, ok agree, maybe better make it as wrapper, I think we will have it on each page where we will have scroll, no?

Reply:
hmm yes, I think so too... i will learn how to do it